src/timeseriesinsights/azext_timeseriesinsights/azext_metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+    "azext.isPreview": false,
+    "azext.isExperimental": true,
+    "azext.minCliCoreVersion": "2.0.67",
+    "azext.maxCliCoreVersion": "3.0.0"
```

Comment:
Is there any special concern to add `maxCliCoreVersion` ?

Reply:
Removed.